### PR TITLE
Add quotes to all python invocations

### PR DESCRIPTION
**Pull request by krdln**

### CODE REVIEW

File: ansible/roles/vmware-template/tasks/deploy.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
     datacenter: "{{ datacenter_name }}"
 
-    uuid: "{{ new_vm.instance.hw_product_uuid }}"
+    id: "{{ new_vm.instance.hw_product_uuid }}"
```
Comment (sodar) [inline ansible/roles/vmware-template/tasks/deploy.yml]: Are you certain that `vmware_guest_facts` accepts `id` as a parameter? According to https://github.com/ansible/ansible/blob/v2.4.0.0-1/lib/ansible/modules/cloud/vmware/vmware_guest_facts.py#L132 it accepts `uuid`, not `id`.

File: vars/unlockNetwork.groovy [hunk 2]
```diff
@@ -10,6 +10,6 @@ def call(String networkName) {
             '--network-name', networkName,
         ]
 
-        sh cmd.join(' ')
+        sh cmd.collect{ arg -> "'" + arg + "'" }.join(' ')
```
Comment (sodar) [inline vars/unlockNetwork.groovy]: Maybe we should move this fragment into separate function? It might be useful in the future.

**Review by sodar — CHANGES_REQUESTED** [review]

File: ansible/roles/vmware-template/tasks/deploy.yml (hunk 1 above)
Comment (krdln) [reply ansible/roles/vmware-template/tasks/deploy.yml]: You're right. I haven't intended to make this change.

File: vars/unlockNetwork.groovy (hunk 2 above)
Comment (krdln) [reply vars/unlockNetwork.groovy]: I've extracted it to `shellCommand` function. Also added escaping according to [this SO answer](https://unix.stackexchange.com/questions/187651/how-to-echo-single-quote-when-using-single-quote-to-wrap-special-characters-in/187654#187654).

**Review by sodar — DISMISSED** [review]

**Comment by krdln** [comment]
@sodar I've added a test for the `shellCommand` helper. What do you think about this approach?

Also, I've let the [gather-build-stats use this helper, and it looks like it works](http://10.84.12.26:8080/job/WinContrail/job/gather-build-stats/30/)

**Review by sodar — APPROVED** [review]

**Pull request merged**